datadog_checks_base/datadog_checks/base/checks/base.py

```diff
@@ -238,6 +238,31 @@ def get_instance_proxy(self, instance, uri, proxies=None):
     def _context_uid(self, mtype, name, tags=None, hostname=None):
         return '{}-{}-{}-{}'.format(mtype, name, tags if tags is None else hash(frozenset(tags)), hostname)
 
+    def submit_histogram_bucket(self, name, value, lower_bound, upper_bound, monotonic, hostname, tags):
```

Comment:
A bit late but I'm thinking that most submit methods dont require the hostname to be passed from the integration side (defaults to '')